src/components/AlertMessage.jsx

```diff
@@ -30,4 +31,9 @@ class AlertMessage extends React.Component {
 	}
 }
 
+AlertMessage.propTypes = {
+	type: PropTypes.string,
+	children:PropTypes.string,
```

Comment:
Alert messages could contain more than just strings. It could also show react elements, numbers, etc.

Try to use a prop type that encompasses all of those possibilities.